index.html

```diff
@@ -122,80 +161,108 @@ <h3>Project name goes here</h3>
-        <li>Ruby on rails</li>
-        <li>RSpec</li>
-        <li>Capibara</li>
-        <li>Selenium</li>
-      </ul>
-      <img class="set-img" src="./assets/Group 18.png">
-    </div>
-    <div class="skills">
-      <h4 class="set-h">Skills</h4>
-      <ul class="set-ul">
-        <li>Codekit</li>
-        <li>Codepan</li>
-        <li>GitHub</li>
-        <li>GitLab</li>
-        <li>Terminal</li>
-      </ul>
-      <img class="set-img" src="./assets/ic_skills.png">
+    <div class="about-col-2">
+      <div class="col-2-child">
+        <div class="languages">
+          <h4 class="set-h">Languages</h4>
+          <ul class="set-ul">
+            <li>JavaScript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+          <img class="set-img" src="./assets/hash.png" />
+        </div>
+        <div class="frameworks">
+          <h4 class="set-h">Frameworks</h4>
+          <ul class="set-ul">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+          <img class="set-img" src="./assets/Group 18.png" />
+        </div>
+        <div class="skills">
+          <h4 class="set-h">Skills</h4>
+          <ul class="set-ul">
+            <li>Codekit</li>
+            <li>Codepan</li>
+            <li>GitHub</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+          <img class="set-img" src="./assets/ic_skills.png" />
+        </div>
+        <div class="dots-3">
+          <img src="./assets/image_dots_with_teal.svg" />
+        </div>
+      </div>
     </div>
   </section>
-  <section class="contact-form">
-    <h2>Get started</h2>
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-    <form method="post" action="https://formspree.io/f/xvoyvapl">
-        <input class="form-items" type="text" name="name" id="full-name" maxlength="30" placeholder="Your name" required>
-        <input class="form-items" type="email" name="email" id="email" placeholder="Your Email" required>
-        <textarea class="form-items" name="text-area" id="text-area" maxlength="500" cols="30" rows="6" placeholder="Write your message here" required></textarea>
+  <section class="contact-form img1" id="contact">
+    <div class="form-ins">
+      <h2>Get started</h2>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat.
+      </p>
+      <form method="post" action="https://formspree.io/f/xvoyvapl">
+        <div class="name-email">
+          <input class="form-items" type="text" name="name" id="full-name" maxlength="30" placeholder="Your name"
```

Comment:
- [ ] I kindly suggest that you make sure your email input is aligned properly to the other inputs like it is in the Figma design.


| Name     | Character |
| ---      | ---       |
| Your Design | <img width="566" alt="image" src="https://user-images.githubusercontent.com/52934128/199773437-6f1582d1-053c-402b-a68c-84a009b0ed25.png">       |
| Figma Design   |  <img width="557" alt="image" src="https://user-images.githubusercontent.com/52934128/199774221-229b4d39-cf3b-4e05-ab84-017e469d6c94.png">  |